examples/graph/main.cpp

```diff
@@ -4,8 +4,8 @@
 #include <QtNodes/StyleCollection>
 
 #include <QtWidgets/QApplication>
-#include <QtGui/QAction>
-#include <QtGui/QScreen>
```

Comment:
I was not able to compile without his change

Reply:
That means you use something prior to 5.15